packages/engine/Source/DataSources/CallbackProperty.js

```diff
@@ -49,11 +50,14 @@ Object.defineProperties(CallbackProperty.prototype, {
 /**
  * Gets the value of the property.
  *
- * @param {JulianDate} time The time for which to retrieve the value.
+ * @param {JulianDate} [time=now] The time for which to retrieve the value. If omitted, now is used.
```

Comment:
I would suggest:

```suggestion
 * @param {JulianDate} [time=JulianDate.now()] The time for which to retrieve the value. If omitted, the current system time is used.
```

just to be a bit more precise.

Reply:
done!